README.md

```diff
@@ -1,139 +1,194 @@
-# Uber Rides Java SDK (beta)
+# Uber Rides Java SDK (Beta)
 
-Official Java SDK (beta) to support the Uber Rides API.
+The SDK helps your Java app interact with the Uber rides API using libraries that wrap the raw HTTP calls made to the Uber API.
 
-## Before you begin
+## Setup
 
-This is a beta Java SDK not intended to be run on Android. An official Android SDK will be released soon.
+### Installing
 
-## Installation
+#### Before you begin
+To use the API, you need to register your app in the [Uber developer dashboard](https://developer.uber.com/dashboard). When you register, the app gets a client ID, secret, and server token used for authentication. 
 
-To use the Uber Rides Java SDK, add the compile dependency with the latest version of the Uber SDK.
+Note: This beta Java SDK is not suitable for Android development. To that end, we will release an official Android SDK soon.
 
-### Gradle
-In the `build.gradle` file:
+#### Gradle
+If using Gradle, add this to your project’s `build.gradle` file:
 ```gradle
 dependencies {
     compile 'com.uber.sdk:rides:0.1.0'
 }
 ```
-### Maven
-In the `pom.xml` file:
+
+#### Maven
+If using Maven, add this to your project’s `pom.xml` file:
 ```xml
 <dependency>
-	<groupId>com.uber.sdk</groupId>
-	<artifactId>rides</artifactId>
-	<version>0.1.0</version>
+  <groupId>com.uber.sdk</groupId>
+  <artifactId>rides</artifactId>
+  <version>0.1.0</version>
 </dependency>
 ```
 
-## Sync v.s. Async
+### Authenticating your app
+
+The oAuth 2.0 authorization flow lets your app request rides on the user’s behalf and access their profile and history without compromising their credentials. It has two parts: First, your app asks the user to authorize and second, it exchanges the authorization code for an access token from Uber. Here is a sample of the authorization flow:
```

Comment:
Make "The oAuth 2.0 authorization flow lets your app request rides on the user’s behalf and access their profile and history without compromising their credentials." more generic because there's more functionality coming. How about "The OAuth 2.0 flow allows you to perform operations on the user's account, such as requesting a ride."